docs/README.rst

```diff
@@ -301,7 +321,7 @@ Backend
 
 Ingress Resources
 -----------------
-The |kctlr-long| supports Kubernetes Ingress resources as an alternative to F5 Resource ConfigMaps.
+The |kctlr-long| supports Kubernetes Ingress resources as an alternative to F5 Resource ConfigMaps and Kubernetes Ingress Resources.
```

Comment:
Did you mean "and OpenShift Route Resources" here?

Reply:
Yes, will fix.